elasticsearch/tests/elasticsearch/elasticsearch.tests.bom

```diff
@@ -41,6 +41,7 @@ brooklyn.catalog:
             id: elasticsearch-cluster
             brooklyn.config:
               cluster.initial.size: 1
+              elasticsearch.bind.address: _site_
```

Comment:
_site_ is an elastic magic word meaning “Any site-local addresses on the system, for example 192.168.0.1.”